plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/config/internal/EssentialsPage.java

```diff
@@ -346,24 +484,23 @@ private void bindContent(DataBindingContext bindingContext) {
 			} else {
 				value.setValue(content);
 			}
-			getWidget().addPropertyChangeListener("content",
-					new PropertyChangeListener() {
+			contentListener = new PropertyChangeListener() {
```

Comment:
Same as for authorListener, this is so that contentListener can be properly disposed.